graylog2-server/src/main/java/org/graylog2/rest/resources/system/indexer/IndicesResource.java

```diff
@@ -173,6 +174,16 @@ public boolean apply(String indexName) {
         return ClosedIndices.create(reopenedIndices, reopenedIndices.size());
     }
 
+    @GET
+    @Timed
+    @Path("/all")
```

Comment:
As I said before, this PR is scoped for a small optimization supporting the web interface. Any further redesign of the API should be done outside of it.